### PR TITLE
QGIS_SERVER_DISABLE_GETPRINT setting environment

### LINKED ISSUE #6042: New QGIS Server setting environement to disable GetPrint and do not load layouts (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#38240
Author: @rldhont
QGIS version: 3.16 (Feature)

**New QGIS Server setting environement to disable GetPrint and do not load layouts**

## Description

Defined a new QGIS Server setting environnement `QGIS_SERVER_DISABLE_GETPRINT` to do not load layouts when QGIS Server reads projects and disable WMS GetPrint request.

This improves projects read time.

Funded by [Ifremer](https://wwz.ifremer.fr/)

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by rldhont**
Goal: Enhance QGIS Server documentation by adding QGIS_SERVER_DISABLE_GETPRINT setting environnement description.

Ticket(s): #6042

### CODE REVIEW

File: docs/server_manual/config.rst [hunk 1]
```diff
@@ -200,6 +200,13 @@ considered as unique without a check.
 
 Do not use it if layers' extent is not fixed during the project's use.
 
+QGIS_SERVER_DISABLE_GETPRINT
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is an option at the project level to improve project read time by disabling the
+layouts load.
+
+This option disables WMS GetPrint request and activates thes QGIS project's flag to do not load layouts.
```
Comment (havatv) [inline docs/server_manual/config.rst:209]:
```suggestion
This is an option at the project level to improve project read time by disabling
loading of layouts.

This option disables the WMS GetPrint request.
Activate this QGIS project flag to not load layouts.
```

**Review by havatv — APPROVED** [review]
Thanks @rldhont. This looks good. I have a suggestion that I hope you will consider.

**Comment by rldhont** [comment]
Thanks @havatv, I have commit your suggestion.

**Comment by havatv** [comment]
Thanks again,  @rldhont!

**Pull request merged**